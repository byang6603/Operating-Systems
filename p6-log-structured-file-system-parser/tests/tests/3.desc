Tests cat of a file in root.
