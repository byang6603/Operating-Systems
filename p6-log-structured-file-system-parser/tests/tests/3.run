../bin/parse_lfs cat /bar.txt ../sample_files/a.img
