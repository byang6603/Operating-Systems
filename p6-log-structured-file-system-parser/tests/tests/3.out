         By CHARLES DICKENS

                                * * * * *

             _With illustrations by Marcus Stone_, _Maurice_
                     _Greiffenhagen_, _and F. Walker_

                                * * * * *

                